electron/index.d.ts

```diff
@@ -1864,12 +1864,7 @@ declare namespace Electron {
 		/**
 		 * Writes data to the clipboard.
 		 */
-		write(data: {
-			text?: string;
-			rtf?: string;
-			html?: string;
-			image?: NativeImage;
-		}, type?: ClipboardType): void;
+		write(data: { text: string; bookmark?: string; } | { rtf: string; } | { html: string; } | { image: NativeImage; }, type?: ClipboardType): void;
```

Comment:
oh, ok, I understood. It's important that one of `text`, `rtf`, `html` or `image` must be specified.
Below must be into compilation error:

```typescript
write({ bookmark: "foo" });
```